### PR TITLE
first pull request for new branch

### PR DESCRIPTION
**Pull request by vivekjkadia**
close Issue

### CODE REVIEW

File: vivekjkadia_test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test file
```
Comment (vivekjkadia) [inline vivekjkadia_test.txt]: First letter should ne capital

**Comment by githubteacher** [comment]
@vivekjkadia :wave: thanks for getting this started! Would you mind adding some info about fun things to do when I visit your hometown? I :heart: 🍣 🍝 and 🗻

**Comment by jwhanson** [comment]
@vivekjkadia Where are the best places to eat?

**Comment by vivekjkadia** [comment]
@jwhanson defiantly in home.lol

**Pull request merged**